tests/testthat/_snaps/verb-joins.md

```diff
@@ -63,16 +63,31 @@
         WHERE (`lf1`.`x1` = `RHS`.`x`)
       )
 
+# multiple joins create a single query
+
+    Code
+      remote_query(out)
+    Output
+      <SQL> SELECT `df3`.`x` AS `x`, `a`, `LHS`.`b` AS `b.x`, `df3`.`b` AS `b.y`
+      FROM (
+        SELECT `df1`.*, `b`
+        FROM `df1`
+        LEFT JOIN `df2`
+          ON (`df1`.`x` = `df2`.`x`)
+      ) `LHS`
+      RIGHT JOIN `df3`
```

Comment:
It seems weird to have a result with multiple queries for a test with title "multiple joins create a single query"